conf/mesos-site.xml

```diff
@@ -89,6 +89,21 @@
     <value>3</value>
   </property>
 
+  <property>
+    <name>mesos.hdfs.jre-url</name>
+    <value>https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz</value>
```

Comment:
Just don't include the property and it will default to the value in HdfsFrameworkConf. In HDFSFrameworkConf you should include this value, i.e. return getConf().get("mesos.hdfs.jre-url", DEFAULT_JRE_URI);